### ISSUE TITLE
Standard <code> element styling violate WCAG 1.4.1 Use of color

### ISSUE THREAD

**Issue by WilcoFiers** [w3c/wai-website-theme#144]
ACT is having difficulties with the styles the WAI site uses for code elements and code blocks.

Here's an example of the issue:
https://www.w3.org/WAI/standards-guidelines/act/rules/aizyf1/proposed/#passed-example-1

![Screenshot Passed example 1 and 2](https://user-images.githubusercontent.com/530687/230438546-6837ac87-80c4-4fb0-af61-85405ecf360b.png)

The contrast between the code and the rest of the text is 1.34:1. It is very easily missed. There is also no other style changes like you see on many other sites. For example code blocks often get an outline, change in background, and a change of font. Any one of these might be sufficient, but more would probably be better.
